src/Router.js

```diff
@@ -3,13 +3,14 @@ import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import Nav from './components/Nav/Nav';
 import Footer from './components/Footer/Footer';
 import Main from './pages/Main/Main';
+import ItemList from './pages/ItemsList/ItemsList';
 
 function Router() {
   return (
     <BrowserRouter>
       <Nav />
       <Routes>
-        <Route path="/" element={<Main />} />
+        <Route path="/itemList" element={<ItemList />} />
```

Comment:
```suggestion
       <Route path="/" element={<Main />} />
        <Route path="/itemList" element={<ItemList />} />
```

기존의 경로는 남겨두어야 추후 발생할 Conflict를 방지할 수 있습니다!

Reply:
완료